src/request/src/middleware/src/storage/index.js

```diff
@@ -14,14 +14,14 @@ export {
   MemoryAdapter
 };
 
-export default class Storage {
+export class Storage {
```

Comment:
Shouldn't this class be in a file of its own, instead of index.js? I'm a fan of one class per file and file names matching class names. And index.js to just be a barrel.